class-00/lab-b/starter-code/app.js

```diff
@@ -184,17 +186,17 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log(joe.scope());
 
 Student.prototype.scopeArrow = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log(joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+// It is undefined.
 // 2. What is "this" when joe.scopeArrow() is invoked?
-//
+// It is the Window root object. 
 // 3. Explain why "this" is different when an arrow function is used.
 //

```

Comment:
What do you understand about the differences of contextual this when using an arrow function vs. the functions you are used to?